drafts/building-github-pull-requests-with-jenkins-pipelines.md

```diff
@@ -0,0 +1,114 @@
+# Building Github Pull Requests with Jenkins Pipelines
+
+We've written about building Github Pull Requests using Jenkins [before](/building-github-pull-requests-with-jenkins/), back in 2013. Now in 2016 we're still strong Jenkins users here at Kabisa and Jenkins has changed quite a bit in recent years. Most notably Jenkins  introduced the notion of [Pipelines](https://jenkins.io/solutions/pipeline/).
```

Comment:
... We've written about building Github Pull Requests [back in 2013](...). Three years later we're still strong Jenkins users here at Kabisa, even though Jenkins changed quite a bit. Most notably it introduced...